src/services/zapper.ts

```diff
@@ -19,6 +20,14 @@ const ZAPPER_AFFILIATE_ADDRESS = "0xFEB4acf3df3cDEA7399794D0869ef76A6EfAff52";
  * tokens and user positions.
  */
 export class ZapperService extends Service {
+  constructor(chainId: ChainId, ctx: Context) {
+    super(chainId, ctx);
+
+    if (![1, 1337].includes(chainId)) {
+      throw new SdkError(`Chain id ${chainId} currently not supported!`);
```

Comment:
How do we handle this error when instantiating the sdk for fantom/arbitrum? Won't it error on init with this? I think the historical precedent is to always be able to instantiate the service, but then if the network is not supported for a method then just return nothing for it

Reply:
Correct, I forgot to address this in this PR after this discussion https://github.com/yearn/yearn-sdk/pull/245#discussion_r825182834

I also realised that it wouldn't let us instantiate for other networks. Will address this shortly

Reply:
Removed the constructor (as it was originally) so that the ZapperService can be instantiated on any chain.

https://github.com/yearn/yearn-sdk/pull/249/commits/7f6403aa241032f37d37018728aa98f4eafeea70